composer.json

```
@@ -10,5 +10,7 @@
             "role": "Maintainer"
         }
     ],
-    "require": {}
+    "require": {
+        "drupal/core": "^8.0 || ^9.0"
```

This can be more elegant (and I also _think_ that this project does not work with core 8.0):
```suggestion
        "drupal/core": "^8.7 || ^9"
```